Examples/Run/Geant4/Common/src/Geant4Common.cpp

```diff
@@ -89,10 +94,12 @@ void setupGeant4Simulation(
   g4Cfg.detectorConstruction = detector.release();
 
   // Set the user actions
-  g4Cfg.runActions = std::move(runActions);
-  g4Cfg.eventActions = std::move(eventActions);
-  g4Cfg.trackingActions = std::move(trackingActions);
-  g4Cfg.steppingActions = std::move(steppingActions);
+  g4Cfg.runAction = runActions.empty() ? nullptr : runActions.front();
```

Comment:
would it make sense to use a generalized the vector of actions and use it here as well?

Reply:
as these are the examples binaries I don't want to touch more code then necessary to ensure that they compile actually and somehow maintain functionality

Reply:
oh yeah makes sense